easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.0_esmumps-intel-2015a.eb

```diff
@@ -0,0 +1,14 @@
+name = 'SCOTCH'
+version = '6.0.0_esmumps'
```

Comment:
no, it's actually a separate version, it's not a variant build, see http://gforge.inria.fr/frs/shownotes.php?release_id=7732

but apparently there's a 6.0.3 instead, so let me switch to that instead